squad-server/plugins/discord-admin-request.js

```diff
@@ -123,9 +131,39 @@ export default class DiscordAdminRequest extends DiscordBasePlugin {
 
     await this.sendDiscordMessage(message);
 
-    await this.server.rcon.warn(
-      info.player.steamID,
-      `An admin has been notified, please wait for us to get back to you.`
-    );
+    await this.server.updatePlayerList;
+    let admins = await this.server.getAdminsWithPermission('canseeadminchat');
+
+    if (this.options.warnInGameAdmins) {
+      for (const player of this.server.players) {
+        if (admins.includes(player.steamID)) {
+          this.amountAdmins++;
+
+          await this.server.rcon.warn(player.steamID, `[${info.player.name}] - ${info.message}`);
+        }
+      }
+
+      if (this.amountAdmins === 0) {
```

Comment:
Again, for cleaner code, as this is a single line inside the if block you can use:
```js
if (this.amountAdmins === 0) await ...
```

The same goes for the else block:
```js
else await ...
```